### PR TITLE
Feature/storage/stg75

### PR DESCRIPTION
**Pull request by rickle-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,13 @@
 ## 12.9.0 (2020-11-11)
 - Fixed a bug where interspersed element types returned by page listing would deserialize incorrectly.
 - Fixed a bug where BlobInputStream would not eTag lock on the blob, resulting in undesirable behavior if the blob was modified in the middle of reading. 
+- Renamed BlobDownloadToFileOptions.rangeGetContentMd5 to BlobDownloadToFileOptions.retrieveContentRangeMd5.
+- Added support for the 2020-04-08 service version. 
+- Added support to upload block blob from URL.
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/CHANGELOG.md:12]: These should be under a new version right?

File: sdk/storage/azure-storage-blob/CHANGELOG.md [hunk 2]
```diff
@@ -7,9 +7,13 @@
...
 - Added support for move and execute permissions on blob SAS and container SAS, and list permissions on blob SAS.
 - Added support to specify a preauthorized user id and correlation id for user delegation SAS.
-- Renamed BlobDownloadToFileOptions.rangeGetContentMd5 to BlobDownloadToFileOptions.retrieveContentRangeMd5
+- Added lease ID parameter to Get and Set Blob Tags.
+- Added blob tags to BlobServiceClient.findBlobsByTags() result.
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/CHANGELOG.md:16]: Same question with these CHANGELOGs

File: sdk/storage/azure-storage-file-share/CHANGELOG.md [hunk 3]
```diff
@@ -13,6 +13,9 @@
 - Renamed setAccessTier to setProperties and deprecated setQuotaInGb in favor of setProperties.
 - Renamed DeleteSnapshotsOptionType to ShareSnapshotsDeleteOptionType in ShareClient.delete
 - Removed ability to create a ShareLeaseClient for a Share or Share Snapshot. This feature has been rescheduled for future release.
+- Added support for the 2020-04-08 service version. 
+- Added support for specifying enabled protocols on share creation
+- Added support for setting root squash on share creation and through set properties.
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-file-share/CHANGELOG.md:18]: Should these go under a new version?

**Review by alzimmermsft — APPROVED** [review]

File: sdk/storage/azure-storage-blob/CHANGELOG.md (hunk 1 above)
Comment (rickle-msft) [reply sdk/storage/azure-storage-blob/CHANGELOG.md:12]: Yea. Good catch.

**Review by gapra-msft — APPROVED** [review]

**Pull request merged**